tests/integration/helpers.bash

```diff
@@ -616,3 +617,36 @@ function requires_kernel() {
 		skip "requires kernel $1"
 	fi
 }
+
+function requires_idmap_fs() {
+	local fs
+	fs=$1
+
+	# We need to true it to avoid CI failure as this binary may return with
```

Comment:
No, it was meant to say "|| true" the command. 

Changed it to:
```
        # We need to "|| true" it to avoid CI failure as this binary may return with
        # something different than 0.
```